helm/memcached/values.yaml

```diff
@@ -42,3 +42,9 @@ memcached:
       repository: giantswarm/bitnami-memcached-exporter
       tag: 0.13.0
       pullPolicy: IfNotPresent
+    ## Resources for metrics MUST be set when using the high-availability architecture with autoscaling enabled
+    ## Otherwise the hpa won't be able to monitor the pod's resources usage
+    resources:
+      requests:
+        memory: 10Mi
+        cpu: 1m
```

Comment:
I'd probably set higher requests here though

Reply:
Well in any case those are only requests so if the container needs more and there is available resources it will get more right ? 
But sure, we could increase those a bit. Any suggestion on a number ?

Reply:
Yes but I'd rather it does not go on rather overloaded node just because requests are so low. I'm not really sure but I'd at least use 50m cpu?